drkafka/src/main/java/com/pinterest/doctorkafka/KafkaClusterManager.java

```diff
@@ -599,9 +599,10 @@ public UnderReplicatedReason getUnderReplicatedReason(String brokerHost,
       List<OutOfSyncReplica> outOfSyncReplicas) {
     Map<TopicPartition, Integer[]> replicasMap = new HashMap<>();
     boolean success = true;
-
-    PriorityQueue<KafkaBroker> brokerQueue = kafkaCluster.getBrokerQueue();
```

Comment:
`brokerQueue` is used in `KafkaCluster.getAlternativeBrokers`, and the priority queue ordering invariant is maintained  in that method.